src/domain/accounts/index.ts

```diff
@@ -11,3 +13,24 @@ export const AccountStatus = {
   Locked: "locked",
   Active: "active",
 } as const
+
+export const checkedCoordinates = ({
+  latitude,
+  longitude,
+}: {
+  latitude: number
+  longitude: number
+}) => {
+  if (latitude < -90 || latitude > 90 || longitude < -180 || longitude > 180) {
+    return new InvalidCoordinatesError()
+  }
+  const coordinates: Coordinates = { latitude, longitude }
+  return coordinates
+}
+
+export const checkedMapTitle = (title: string) => {
+  if (title.length < 3 || title.length > 100) {
+    return new InvalidTitleLengthError()
+  }
+  return title as BusinessMapTitle
```

Comment:
@bodymindarts is that how it's supposed to be cast in this instance?

Reply:
Yes - inside the `checkedToXxx` function is the only place where casting like this is allowed.